torchreid/models/common.py

```diff
@@ -39,6 +39,9 @@ def __init__(self,
     def is_classification(self):
         return self.type == 'classification' or self.type == 'multilabel'
 
+    def get_num_features(self):
+        return self.num_features
```

Comment:
`num_features` is not defined here. It could be passed via constructor as the other common fields

Reply:
I can pass feature_dim as the default